docs/user_docs/kubeblocks-for-apecloud-mysql/cluster-management/create-and-connect-an-apecloud-mysql-cluster.md

```diff
@@ -46,49 +95,212 @@ This tutorial shows how to create and connect to an ApeCloud MySQL cluster.
+   ```
 
-:::
+2. Verify whether this cluster is created successfully.
+
+   ```bash
+   kbcli cluster list -n demo
+   >
+   NAME        NAMESPACE   CLUSTER-DEFINITION   VERSION           TERMINATION-POLICY   STATUS    CREATED-TIME
+   mycluster   demo        apecloud-mysql       ac-mysql-8.0.30   Delete               Running   Sep 19,2024 16:01 UTC+0800
+   ```
+
+</TabItem>
+
+<TabItem value="kubectl" label="kubectl">
+
+1. Create an ApeCloud MySQL cluster.
+
+   KubeBlocks implements a `Cluster` CRD to define a cluster. Here is an example of creating a RaftGroup Cluster.
+
+   If you only have one node for deploying a RaftGroup Cluster, set `spec.affinity.topologyKeys` as `null`. But for a production environment, it is not recommended to deploy all replicas on one node, which may decrease the cluster availability.
+
+   ```yaml
+   cat <<EOF | kubectl apply -f -
+   apiVersion: apps.kubeblocks.io/v1alpha1
+   kind: Cluster
+   metadata:
+     name: mycluster
+     namespace: demo
+   spec:
+     clusterDefinitionRef: apecloud-mysql
+     clusterVersionRef: ac-mysql-8.0.30
+     terminationPolicy: Delete
+     affinity:
+       podAntiAffinity: Preferred
+       topologyKeys:
+       - kubernetes.io/hostname
+     tolerations:
+       - key: kb-data
+         operator: Equal
+         value: 'true'
+         effect: NoSchedule
+     componentSpecs:
+     - name: mysql
+       componentDefRef: mysql
+       enabledLogs:
+       - error
+       - general
+       - slow
+       disableExporter: true
+       replicas: 3
+       serviceAccountName: kb-acmysql-cluster
+       resources:
+         limits:
+           cpu: '0.5'
+           memory: 0.5Gi
+         requests:
+           cpu: '0.5'
+           memory: 0.5Gi
+       volumeClaimTemplates:
+       - name: data
+         spec:
+           accessModes:
+           - ReadWriteOnce
+           resources:
+             requests:
+               storage: 20Gi
+   EOF
+   ```
+
+   | Field                                 | Definition  |
+   |---------------------------------------|--------------------------------------|
+   | `spec.clusterDefinitionRef`           | It specifies the name of the ClusterDefinition for creating a specific type of cluster.  |
+   | `spec.clusterVersionRef`              | It is the name of the cluster version CRD that defines the cluster version.  |
+   | `spec.terminationPolicy`              | It is the policy of cluster termination. The default value is `Delete`. Valid values are `DoNotTerminate`, `Halt`, `Delete`, `WipeOut`.  <p> - `DoNotTerminate` blocks deletion operation.  </p><p> - `Halt` deletes workload resources such as statefulset and deployment workloads but keep PVCs.  </p><p> - `Delete` is based on Halt and deletes PVCs. </p><p> - `WipeOut` is based on Delete and wipe out all volume snapshots and snapshot data from a backup storage location. </p>|
```

Comment:
The `Halt` option has been deleted in kb v1.0, so it should be removed if this doc is orienting to that version.

Reply:
These docs are still based on v0.9.0. This PR is created to merge the tutorials in API docs into user docs.